src/lib/components/bottom-nav.svelte

```diff
@@ -27,6 +28,33 @@
   let viewNamespaces = writable(false);
   let viewSettings = false;
 
+  const escHandler = new AbortController();
```

Comment:
What is the purpose of this? I see it added in the event listener, but where is it referenced, in a `fetch` somewhere?

Reply:
You can use abortControllers to remove event listeners now. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#signal